content/docs/aws/authn-authz.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Authentication and Authorization"
+description = "Authentication and Authorization support for Kubeflow in AWS"
+weight = 90
++++
+
+This section shows the how to setup Kubeflow with Authentization and Authorization support in Amazon Web Services (AWS).
+
+
+## Manifest
+
+In order to simply your setups, we highly recommend you to use this manifest.
+[kfctl_aws_cognito.yaml](https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_aws_cognito.yaml)
```

Comment:
Should this refer to the version-specific file name? 
`{{% config-uri-aws-cognito %}}`

Reply:
Yes. Change to use config uri